feincms_extensions/content_types.py

```diff
@@ -0,0 +1,50 @@
+from feincms.content.medialibrary.models import MediaFileContent
+from feincms.content.richtext.models import RichTextContent
+from feincms.content.section.models import SectionContent
+
+
+class JsonRichTextContent(RichTextContent):
+    class Meta(RichTextContent.Meta):
+        abstract = True
+
+    def json(self, **kwargs):
+        """Return a json serializable dictionary containing the content."""
+        return {'html': self.text}
+
+
+class JsonSectionContent(SectionContent):
+    class Meta(SectionContent.Meta):
+        abstract = True
+
+    def json(self, **kwargs):
+        """Return a json serializable dictionary containing the content."""
+        mediafile = self.mediafile
+        return {
+            'title': self.title,
+            'html': self.richtext,
+            'mediafile': {
+                'url': mediafile.file.url,
+                'type': mediafile.type,
+                'created': mediafile.created,
+                'copyright': mediafile.copyright,
+                'file_size': mediafile.file_size,
+            },
```

Comment:
Looks like a `mediafile` could be [`blank` or `null`](https://github.com/feincms/feincms/blob/53ec56bea98cd7d9c1b58e5dd753d467106a3eb5/feincms/content/section/models.py#L37-L39). Will it fail when accessing these attributes?